data/settings/1.44.x/container-runtime-plugins.toml

```diff
@@ -0,0 +1,137 @@
+
+[[docs.ref.soci-snapshotter-pull-mode]]
+name_override = "soci-snapshotter.pull-mode"
+description = "Pull mode for SOCI snapshotter."
+default = "`parallel-pull-unpack`"
+accepted_values = [
+    "`parallel-pull-unpack`"
+]
+see = [
+    ["[SOCI Snapshotter Configuration](https://github.com/awslabs/soci-snapshotter/blob/main/docs/parallel-mode.md#configuration)"]
+]
+tags = ["soci-parallel-pull-configuration"]
+
+[[docs.ref.soci-snapshotter-pull-mode.example]]
+value = "\"parallel-pull-unpack\""
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-downloads]]
+name_override = "soci-snapshotter.parallel-pull-unpack.max-concurrent-downloads"
+description = "Total concurrent downloads across all images."
+default = "`-1` (unbounded)"
+accepted_values = [
+    "`-1` for unbounded",
+    "positive integer"
+]
+see = [
+    ["[SOCI Snapshotter Configuration](https://github.com/awslabs/soci-snapshotter/blob/main/docs/parallel-mode.md#configuration)"]
+]
+tags = ["soci-parallel-pull-configuration"]
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-downloads.example]]
+value = "10"
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-downloads-per-image]]
+name_override = "soci-snapshotter.parallel-pull-unpack.max-concurrent-downloads-per-image"
+description = "Concurrent downloads per individual image."
+default = "`3`"
+accepted_values = [
+    "positive integer"
+]
+see = [
+    ["[SOCI Snapshotter Configuration](https://github.com/awslabs/soci-snapshotter/blob/main/docs/parallel-mode.md#configuration)"]
+]
+tags = ["soci-parallel-pull-configuration"]
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-downloads-per-image.example]]
+value = "5"
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-concurrent-download-chunk-size]]
+name_override = "soci-snapshotter.parallel-pull-unpack.concurrent-download-chunk-size"
+description = "Size of download chunks."
+default = "`unlimited` (unbounded)"
+accepted_values = [
+    "`unlimited` for unbounded",
+    "byte size string (e.g. 1024, 1kb, 3mib, 2gb)"
+]
+see = [
+    ["[SOCI Snapshotter Configuration](https://github.com/awslabs/soci-snapshotter/blob/main/docs/parallel-mode.md#configuration)"]
+]
+tags = ["soci-parallel-pull-configuration"]
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-concurrent-download-chunk-size.example]]
+value = "\"8mb\""
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-unpacks]]
+name_override = "soci-snapshotter.parallel-pull-unpack.max-concurrent-unpacks"
+description = "Total concurrent unpack operations."
+default = "`-1` (unbounded)"
+accepted_values = [
+    "`-1` for unbounded",
+    "positive integer"
+]
+see = [
+    ["[SOCI Snapshotter Configuration](https://github.com/awslabs/soci-snapshotter/blob/main/docs/parallel-mode.md#configuration)"]
+]
+tags = ["soci-parallel-pull-configuration"]
+
+[[docs.ref.soci-snapshotter-parallel-pull-unpack-max-concurrent-unpacks.example]]
+value = "5"
```

Comment:
The example says
```
[settings.container-runtime-plugins]
soci-snapshotter-parallel-pull-unpack-max-concurrent-downloads-per-image = 5
```
I think its supposed to be 
```
[settings.container-runtime-plugins]
soci-snapshotter.parallel-pull-unpack.max-concurrent-downloads-per-image = 5
```

This error is there in other examples as well.

Reply:
Ah great catch; I'll fix that